core/src/com/unciv/logic/civilization/TechManager.kt

```diff
@@ -30,6 +30,7 @@ class TechManager {
     var freeTechs = 0
     /** For calculating Great Scientist yields - see https://civilization.fandom.com/wiki/Great_Scientist_(Civ5)  */
     var scienceOfLast8Turns = IntArray(8){0}
+    var getScienceFromResearchAgreement = 0
```

Comment:
Please rename this to scienceFromResearchAgreements, 
getScience...etc is a name for a *function* that returns an amount